pages/getting-started/installing-cypress.md

```diff
@@ -0,0 +1,334 @@
+
+## Linux
+
+Se você estiver usando Linux, você precisará das dependências necessárias instaladas no seu sistema.
+
+Nós também temos uma versão oficial cypress/base em container docker com todas as dependências necessárias instaladas.
+
+### Ubuntu/Debian
+```
+apt-get install libgtk2.0-0 libgtk-3-0 libgbm-dev libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xauth xvfb
+```
+
+### CentOS
+```
+yum install -y xorg-x11-server-Xvfb gtk2-devel gtk3-devel libnotify-devel GConf2 nss libXScrnSaver alsa-lib
+```
+
+# Instalando
+
+## `npm install`
+
+Instale Cypress usando `npm`:
+```
+cd /caminho/do/diretorio
+npm install cypress --save-dev
+```
+
+Isto vai instalar o Cypress localmente como uma dependência de desenvolvimento em seu projeto.
+
+> Tenha certificado que você já tenha rodado `npm init` ou tenha o diretório `node_modules` ou o arquivo `package.json` no diretório raiz do seu projeto para garantir que o cypress foi instalado no diretório correto.
+
+> Note que o pacote npm do cypress é um wrapper que envolve o binário do Cypress. A versão do pacote npm determina qual versão do binário baixado. Como a versão 3.0, o binário baixado como um diretório de cache global para ser utilizado entre os projetos.
+
+> Boa prática
+
+> A abordagem recomendada é de instalar o Cypress usando o `npm` porque: 
+    - Cypress é versionada como as outras dependências.
+    - É muito simples rodar o Cypress na Integração Contínua.
+
+## `yarn add`
+
+Instalando o cypress usando o `yarn`:
+
+```
+cd /your/project/path
+yarn add cypress --dev
+```
+
+## download direto
+
+Se você não estiver utilizando Node ou npm no seu projeto ou você quer tentar usar o Cypress rapidamente, você sempre poderá realizar o download do Cypress diretamente do nosso CDN. 
+
+> Gravando as execuções para o Dashboard não é possível pelo download direto. Este download tem como oferecer uma forma rápida de utilizar o Cypress. Para gravar os testes no Dashboard, você precisará instalar o Cypress como uma depedência npm.
+
+O download direto sempre trará a última versão disponível. Sua plataforma será identificada automaticamente.
+
+Você pode descompactar manualmente e clicar duas vezes. Cypress irá ser executado sem precisar instalar nenhuma dependência.
+
+## Integração contínua
+
+Por favor leia nossa documentação sobre [Integração Contínua](#) para ajudar na instação do Cypress na CI (Integração Contínua). Quando estiver rodando linux você precisa'ra instalar algumas dependências de sistema ou você pode usar a nossa imagem docker que possui tudo que você precisa.
+
+# Abrindo cypress
+
+Se você usou `npm` para instalação, Cypress foi instalado no diretório `./node_modules`, com seus binários executáveis accessíveis em `./node_modules/.bin`.
+
+Agora você pode abrir o Cypress do seu projeto principal com uma das seguintes formas:
+
+A forma longa com o caminho completo
+```
+./node_modules/.bin/cypress open
+```
+
+Ou com o atalho usando `npm bin`
+```
```

Comment:
Oops, it seems there something missing here.

```
$(npm bin)/cypress open
```

Reply:
done